src/Utils/Validate.js

```diff
@@ -4,8 +4,11 @@ const Required = (value) => {
   return 'Required!';
 };
 
-const RequiredInput = () => {
-  return false;
+const isURLValid = (value) => {
+  const regex = new RegExp('^$|([Hh][Tt][Tt][Pp]|[Ff][Tt][Pp])([Ss])?://.+$');
+  const isTrue = regex.test(value);
+  if (isTrue) return undefined;
+  return 'Invalid URL';
```

Comment:
It might be a good idea to use `<FormattedMessage>` here, since it goes to UI and could be translated